src/lib/utils.svelte.test.ts

```
@@ -0,0 +1,108 @@
+
+			expect(formatted).toContain('Dec');
+			expect(formatted).toContain('25');
+		});
+	});
+
+	describe('downloadQRCode()', () => {
+		let createElementSpy: ReturnType<typeof vi.spyOn>;
+		let appendChildSpy: ReturnType<typeof vi.fn>;
+		let removeChildSpy: ReturnType<typeof vi.fn>;
+		let clickSpy: ReturnType<typeof vi.fn>;
+		let mockAnchor: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)